docs/en/operations/table_engines/custom_compression_codec.md

```diff
@@ -0,0 +1,44 @@
+
+# Column compression codecs
+
+Besides default data compression, defined in [server settings](../server_settings/settings.md#compression), per-column specification is also available.
+
+Supported compression algorithms:
+
+- `NONE` - no compression for data applied
+- `LZ4`
+- `LZ4HC(level)` - (level) - LZ4 compression algorithm with defined level.
```

Comment:
Need to add information about default levels: LZ4HC -- 9, ZSTD -- 1, Delta -- sizeof(type) if type size is 1, 2, 4, 8 and 1 in other cases.

Reply:
>  - `LZ4HC(level)` - >>**(level)**<< - LZ4 compression algorithm with defined level. 

typo?